src/Admin/AdminUser.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EtoA\Admin;
+
+class AdminUser
+{
+	public ?int $id = null;
+	public ?string $passwordString;
+	public bool $forcePasswordChange = false;
+	public string $nick = "";
+	public string $name = "";
+	public string $email = "";
+	public string $tfaSecret = "";
+	public int $playerId = 0;
+	public string $boardUrl = "";
+	public string $userTheme = "";
+	public bool $ticketEmail = false;
+	public bool $locked = false;
+	public bool $isContact = true;
+	public array $roles = [];
+
+	function __toString()
```

Comment:
I would prefer not to use `__toString`. Using string operations on objects is not something that should be done.

Reply:
done 96d77cd